### ISSUE TITLE
Support .env.prod

### ISSUE THREAD

**Issue by fabiospampinato** [mikestead/vscode-dotenv#14]
Since `.env.dev` and `.env.stag` are explicitly supported I think `.env.prod` should be supported too.

**Comment by jaydenseric** [comment]
Also, Zeit Now supports `.env.build`:

https://zeit.co/docs/v2/environment-variables-and-secrets#local-development

Instead of trying to whitelist know variations, it's easier to just do:

```json
{
  "files.associations": {
    "*.env.*": "dotenv"
  }
}
```

**Comment by sampl** [comment]
Yes please! @jaydenseric's suggestion also supports files I have `.env.stage` `.env.live` etc etc

**Comment by mikestead** [comment]
@jaydenseric when contributing a [language extension](https://code.visualstudio.com/api/references/contribution-points#contributes.languages) you need to provide the explicit extensions from what I've tested (I tried pattern matching to begin with). 

What you're talking about is something I've described in the readme around vscode settings, but I don't see a way to do this in an extension itself.

Are you able to provide more details / references of how to achieve this in an extension, or perhaps your comment was aimed at @fabiospampinato and not the extension itself?

**Comment by GorvGoyl** [comment]
meanwhile, can we please have support for `.env.prod` , `.env.build` until we figure out the way to whitelist based on patterns?

**Comment by GorvGoyl** [comment]
lol.. I'm not proud of it but we can set language mod for *.prod file as python or makefile to get syntax highlighting and # comment support. 

```
"files.associations": {
    ".env.prod": "python"  // "makefile"
  }
```

**Comment by tracker1** [comment]
@jaydenseric thank you... just did ".env.*" for myself...  I generally only commit a `.env.sample` which works for local/development, but must explicitly copy to `.env` (git ignored) local to run... other environments never committed, never set.
